lib/screens/mobile/sole_sphere.dart

```diff
@@ -0,0 +1,158 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_internship/data/sneakers_list.dart';
+import 'package:flutter_internship/helpers/mobile_text.dart';
+import 'package:flutter_internship/widgets/components/discount_card.dart';
+import 'package:flutter_internship/widgets/components/product_card.dart';
+import 'package:flutter_internship/widgets/components/product_card_special.dart';
+import 'package:flutter_internship/screens/mobile/sneakers_grid.dart';
+import 'package:flutter_svg/svg.dart';
+
+class SoleSphere extends StatelessWidget {
+  const SoleSphere({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      color: const Color(0xFFF8F8F8),
+      child: Column(
+        crossAxisAlignment: CrossAxisAlignment.start,
+        children: [
+          _SoleSphereTitle(),
+          Expanded(
+            child: SingleChildScrollView(
+              child: Column(
+                children: [
+                  const MobileText(text: 'Best deal'),
+                  const Padding(
+                    padding: EdgeInsets.symmetric(horizontal: 20),
+                    child: ProductCardSpecial(
+                      image: 'lib/assets/images/pngwing 2.png',
+                      title: 'NIKE Air Jordan',
+                      description: 'Grab the last pair now',
+                      price: '160.00 €',
+                    ),
+                  ),
+                  Row(
+                    mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                    children: [
+                      const MobileText(text: 'Sneakers'),
+                      TextButton(
+                        onPressed: () {
+                          Navigator.push(
+                            context,
+                            MaterialPageRoute(
+                              builder: (context) => const SneakersGrid(),
+                            ),
+                          );
+                        },
+                        child: const BlueText(text: 'View all'),
+                      ),
+                    ],
+                  ),
+                  Padding(
+                    padding: const EdgeInsets.only(left: 20),
+                    child: _buildProductListView(context),
+                  ),
```

Comment:
Never use function to build Widgets. We talked about it and what is cons for that. There is still padding on left side when scrolling, see image.
<img width="562" alt="Screenshot 2024-01-30 at 09 37 20" src="https://github.com/damjan-Igrowth/flutter-internship/assets/119666343/a4ef762a-5f8e-4e7d-95d7-2f79115d0d86">